src/colmap/estimators/cost_functions.h

```diff
@@ -349,182 +349,192 @@ inline void EigenQuaternionToAngleAxis(const T* eigen_quaternion,
   ceres::QuaternionToAngleAxis(quaternion, angle_axis);
 }
 
-// 6-DoF error on the absolute pose. The residual is the log of the error pose,
-// splitting SE(3) into SO(3) x R^3. The 6x6 covariance matrix is defined in the
-// reference frame of the camera. Its first and last three components correspond
-// to the rotation and translation errors, respectively.
-struct AbsolutePoseErrorCostFunctor {
+// 6-DoF error on the absolute camera pose. The residual is the log of the error
+// pose, splitting SE(3) into SO(3) x R^3. The 6x6 covariance matrix is defined
+// in the reference frame of the camera. Its first and last three components
+// correspond to the rotation and translation errors, respectively.
+struct AbsolutePosePriorCostFunctor {
  public:
-  AbsolutePoseErrorCostFunctor(const Rigid3d& cam_from_world,
-                               const EigenMatrix6d& covariance_cam)
-      : world_from_cam_(Inverse(cam_from_world)),
-        sqrt_information_cam_(SqrtInformation(covariance_cam)) {}
+  AbsolutePosePriorCostFunctor(const Rigid3d& cam_from_world_prior,
+                               const Eigen::Matrix6d& cam_from_world_prior_cov)
```

Comment:
`cam_from_world_prior_cov` does not make it clear in which reference frame the covariance is defined, how about `cam_prior_cov` instead?

Reply:
How about cam_cov_from_world_prior ?

Reply:
Equivalently i_cov_from_j_prior in the other instance.

Reply:
Sure, this works too.